src/orion/algo/tpe.py

```diff
@@ -308,47 +337,51 @@ def suggest(self, num=None):
         num: int, optional
             Number of trials to sample. If None, TPE will sample all random trials at once, or a
             single trial if it is at the Bayesian Optimization stage.
-        :param num: how many sets to be suggested.
 
         .. note:: New parameters must be compliant with the problem's domain
            `orion.algo.space.Space`.
         """
+        # TODO: Code below assumes that `num` is an int, but docstring says it can be None.
```

Comment:
Doc is out-of-date. You can correct docstring and remove the assert.

Reply:
Fixed in 5bce0b8a430543624c811628b28b9f096cec9c64